### DISCUSSION TITLE
Deck Zoom Level To Altitude Meters?

### DISCUSSION THREAD

**Discussion by jfuehner** [visgl/deck.gl#5719] [Q&A] [answered]
Does deck have a way to convert from zoom level to approx altitude in meters? Or would there be a calculation on deck's zoom level that could be performed?

I am currently using nebula.gl for drawing. All GeoJSON values however are returned with lat/lon only values. I would like to add the third/option altitude value to the GeoJSON being returned.

**Comment by Pessimistress** [comment, marked as answer]
What altitude are you trying to get? The altitude of the camera? What does that have to do with the altitude of GeoJSON features?

**Reply by jfuehner** [reply]
Trying to get/set the altitude of the camera at the time the objects are drawn.

I would like to be able to redraw/visualize the objects at at the altitude they were drawn at as opposed to always being clamped to the ground.

**Reply by Pessimistress** [reply]
That's not how camera works. The camera is looking at the ground. It cannot see things at the same altitude of itself.
